src/commands.c

```diff
@@ -222,7 +223,7 @@ unsigned int CommandSpecGetAggregateFlags(RaftRedisCommandArray *array, unsigned
         const char * cmd_str = RedisModule_StringPtrLen(cmd, &cmd_len);
 
         if (!strncasecmp("asking", cmd_str, cmd_len)) {
-            /* TODO: when asking <magic> support is added, need to adjust to argv[2] */
+            /* TODO: when asking <migration_session_key> support is added, need to adjust to argv[2] */
```

Comment:
Why do we do something specific to migration and asking (on line 229) here? Is this leftover?

Reply:
Don't think it's leftover - in case of asking cmd, we should take the cmd name from argv[1] (in the current implementation), but agree we can remove this comment, because changing the ASKING spec is related to redis, and won't change in the nearest future.

Reply:
Agree, will remove it in the better-asking PR.